src/Function.h

```diff
@@ -211,6 +213,15 @@ class Function {
      * buffer_t type. */
     EXPORT bool extern_definition_uses_old_buffer_t() const;
 
+    /** Get the proxy Expr for the extern stage. This is an expression
+     * known to have the same data access pattern as the extern
+     * stage. For most Functions, including those with extern
+     * definitions, this will be an undefined Expr. */
+    // @{
+    EXPORT Expr extern_definition_proxy_expr() const;
```

Comment:
Is it possible to be a bit more explicit re: what "same data access pattern" implies re: requirements on this Expr? E.g. does it mean the bounds of reads from inputs and writes to outputs from the extern stage are no larger than what bounds inference will infer? Clearly it is not possible to make the Expr and the extern have exactly the same set of loads and stores...

Reply:
Exactly the same is the current meaning. It could be relaxed to be a statement about the bounds. I'll tweak the comment.